<launch>
  <node name="robot_command Server" pkg="robot_command" type="Server" output="screen">
    <param name="roboter_ip" value="127.0.0.1:29999" type="str" />
    <param name="roboter_port" value="29999" type=int />
    <param name="program_name" value="program.urp" type="str" />
  </node>
</launch>

